app/src/main/java/com/example/touchcar/presentation/main_menu/recycler/MainMenuAdapter.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.touchcar.presentation.main_menu.recycler
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.example.touchcar.databinding.ManufacturerRecyclerItemBinding
+import com.example.touchcar.domain.entity.Manufacturer
+import java.util.*
+
+class MainMenuAdapter(private val onItemClickListener: (Manufacturer) -> Unit) : RecyclerView.Adapter<MainMenuViewHolder>() {
+
+    var manufacturers: List<Manufacturer> = Collections.emptyList()
```

Comment:
В котлине есть свой метод короткий
```suggestion
    var manufacturers: List<Manufacturer> = emptyList()
```